### ISSUE TITLE
Enhance Configuration/Typo3ConfVars/DB to hint at other possible configuration keys

### ISSUE THREAD

**Issue by sgrossberndt** [TYPO3-Documentation/TYPO3CMS-Reference-CoreApi#2252]
Looking at https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/Configuration/Typo3ConfVars/DB.html it seems these are the only available configuration keys for `Configuration/Typo3ConfVars/DB`, but this is not the case. There is `wrapperClass` and additionally 
lots of options from doctrine.
There is another page in the documentations which explains this:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Database/Configuration/Index.html#configuration
At least a hint to this page should be added, even better would be explicit hints at the available keys in the current table format.

**Comment by linawolf** [comment]
I agree, can you provide a PR, @sgrossberndt ?

**Comment by brotkrueml** [comment]
@sgrossberndt @linawolf As I updated the DB page currently, I looked to get the available parameters onto that page. By now, I can only see the `ssl` parameters for mysqli/postgresql which are missing (on both pages you referenced).

I think we should add just a link to the doctrine page from those pages. The most common parameters are described in our docs, for special cases look up the doctrine docs. This way we don't have to check the docs every time doctrine releases a new version (we have 2.13 for v11, 3.5 for v12 and 4.0 is in beta, maybe also v12 or not before v13?).
